core/domain/story_jobs_one_off_test.py

```diff
@@ -138,13 +138,15 @@ def test_migration_job_converts_old_story(self):
         """
         # Generate story with old(v1) story contents data.
         self.save_new_story_with_story_contents_schema_v1(
-            self.STORY_ID, self.albert_id, 'A title',
+            self.STORY_ID, 'image.svg', self.albert_id, 'A title',
```

Comment:
Shouldn't there be some test to ensure that story_contents migrates correctly (i.e. it is in one format before the migration and is in a different format after the migration)?